src/Workspaces/Core/Portable/FindSymbols/FindReferences/Finders/PropertySymbolReferenceFinder.cs

```diff
@@ -171,8 +172,9 @@ private static bool IsForEachProperty(IPropertySymbol symbol)
                     }
 
                     var location = argumentList.SyntaxTree.GetLocation(new TextSpan(argumentList.SpanStart, 0));
+                    var valueUsageInfo = semanticModel.GetValueUsageInfo(node, semanticFacts, cancellationToken);
```

Comment:
curious, why semanticFacts is needed?

Reply:
It will not be needed once https://github.com/dotnet/roslyn/pull/30123 is merged. For now, we use it for bunch of helpers.